src/Form.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
```

Comment:
@fchen829 , please write a test file `Form.test.js` for this Form component. I recommend you to save it in `/tests` folder because this is what Prof wanted us to do. You can create a new folder named `frontend` inside so as to save your test files.